packages/interactivity/src/index.ts

```diff
@@ -38,6 +38,10 @@ export {
 
 export { useState, useRef } from 'preact/hooks';
 
+if ( globalThis.SCRIPT_DEBUG ) {
+	import( 'preact/debug' );
```

Comment:
The `preact/debug` import needs to be static and be [the first import of the application](https://preactjs.com/guide/v10/debugging/#:~:text=Make%20sure%20that%20this%20import%20is%20the%20first%20import%20in%20your%20whole%20app) so I'm afraid we can't do this.